bats/tests/preferences/surface-invalid-args.bats

```diff
@@ -0,0 +1,61 @@
+load '../helpers/load'
+
+local_setup() {
+    if is_windows; then
+        skip "test not applicable on Windows"
+    fi
+}
+
+@test 'initial factory reset' {
+    factory_reset
+}
+
+can_set_vm_to_vz() {
+    if ! is_macos; then
+        false
+    else
+        version=$(/usr/bin/sw_vers -productVersion)
+        majorMinorVersion="${version%.*}"
+        majorVersion="${majorMinorVersion%.*}"
+        minorVersion="${majorMinorVersion#*.}"
+        if ((majorVersion >= 14)); then
+            true
+        elif ((majorVersion <= 12)); then
+            false
+        elif ((minorVersion >= 3)); then
+            true
+        else
+            case "$(uname -m)" in
+            x86_64) true ;;
+            *) false ;;
+            esac
+        fi
+    fi
+}
+
+@test 'mac-specific failure for unacceptable start setting' {
+    if ! is_macos; then
+        skip 'need a mac for the --experimental.virtual-machine.type setting'
+    elif can_set_vm_to_vz; then
+        skip 'no error setting experimental.virtualMachine.type to "vz" on this platform'
+    fi
+    # Don't use launch_the_application so we can check non-dev-mode error messages
+    if using_dev_mode; then
+        yarn dev --experimental.virtualMachine.type vz --no-modal-dialogs &
+    else
+        rdctl start --experimental.virtual-machine.type vz --no-modal-dialogs
+    fi
```

Comment:
I don't like this at all and think it should all be abstracted by `launch_application`, but agree that this is not the time to make those changes.

Reply:
This was needed when `rdctl start` was duplicating the error-checking done by the backend. But it isn't any longer, and calling `launch_the_application` is fine.